module/solutions/helper.js

```diff
@@ -2992,4 +2995,116 @@ module.exports = class SolutionsHelper {
 			}
 		})
 	}
+
+	/**
+	 * Fetch template observation based on solution Id.
+	 * @method
+	 * @name details
+	 * @param {String} solutionId - Solution Id.
+	 * @returns {Object} - Details of the solution.
+	 */
+
+	static details(solutionId, bodyData = {}, userId = '', userToken = '') {
```

Comment:
@Prajwal17Tunerlabs why do you need userToken here?

Reply:
unused token removed form here